### DISCUSSION TITLE
Ads popping up

### DISCUSSION THREAD

**Discussion by helven0** [MakeYourLifeEasier/novel-ebook.com#176] [Q&A] [answered]
When i loaded the page to download some novel ads started poping up .
Before the site was clean, now there are ads or is there viruse on my phone showings those ads ?
Plz answer why were ads added and will they be removed?

**Comment by MichaelGuardian** [comment, marked as answer]
I would love to remove the ads, believe me. But with the latest update server cost literally quadrupled and I just don't want to pay for out of my own pocket, especially since I don't even use my site anymore. If the community can somehow pay 25$ a month I would remove the ads without hesitation.

Fun fact: Even with the ads, I still have to cover some of the cost with my own money.
